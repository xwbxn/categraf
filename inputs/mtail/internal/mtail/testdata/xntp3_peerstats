54695 7690.466 64.113.32.5 93b4 0.002345 0.01001 0.00090
