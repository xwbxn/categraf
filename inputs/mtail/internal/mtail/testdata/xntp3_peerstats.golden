gauge peer_status {peer=64.113.32.5} 18 2008-08-17T02:08:10Z
gauge peer_select {peer=64.113.32.5} 3 2008-08-17T02:08:10Z
gauge peer_count {peer=64.113.32.5} 11 2008-08-17T02:08:10Z
gauge peer_code {peer=64.113.32.5} 4 2008-08-17T02:08:10Z
gauge peer_offset {peer=64.113.32.5} 0.002345 2008-08-17T02:08:10Z
gauge peer_delay {peer=64.113.32.5} 0.01001 2008-08-17T02:08:10Z
gauge peer_dispersion {peer=64.113.32.5} 0.00090 2008-08-17T02:08:10Z
counter num_peerstats {peer=64.113.32.5} 1 2008-08-17T02:08:10Z
